warp10/src/main/java/io/warp10/script/mapper/MapperTick.java

```diff
@@ -51,6 +52,10 @@ public Object apply(Object[] args) throws WarpScriptException {
     long[] elevations = (long[]) args[5];
     Object[] values = (Object[]) args[6];
 
+    if (0 == values.length) {
+      return new Object[] {0L, GeoTimeSerie.NO_LOCATION, GeoTimeSerie.NO_ELEVATION, null};
```

Comment:
The bucket is empty regarding its value, but it does have a tick. Are we certain we don't want to return that ?

Reply:
Yes. A use case of mapper.tick is to compute data loss  (mapper.tick followed by mapper.delta).